tests/system/action/motion_category/test_create.py

```diff
@@ -0,0 +1,43 @@
+from tests.system.base import BaseSystemTestCase
+from tests.util import get_fqid
+
+
+class MotionCategorySystemTest(BaseSystemTestCase):
+    def test_create(self) -> None:
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion_category.create",
+                    "data": [{"name": "test_Xcdfgee"}],
+                }
+            ],
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assert_model_exists(get_fqid("motion_category/1"))
+        model = self.datastore.get(get_fqid("motion_category/1"))
+        assert model.get("name") == "test_Xcdfgee"
+
+    def test_create_empty_data(self) -> None:
+        response = self.client.post(
+            "/", json=[{"action": "motion_category.create", "data": [{}]}],
+        )
+        self.assertEqual(response.status_code, 400)
+        self.assertIn(
+            "data[0] must contain [\\'name\\'] properties", str(response.data),
```

Comment:
Personally, I find these error message checks unnecessary, since they are not relevant for the funcionality of the service and if e.g. fastjsonschema decides to change its error messages you have to adjust all tests. But that's just my opinion, Norman clearly has another one, feel free to build your own :D

Reply:
These checks are okay so far, if fastjsonschema changes its error messages too often, we can rethink it.